example/flutter_example/lib/book.dart

```diff
@@ -0,0 +1,33 @@
+class PersonDto {

```

Comment:
There are multiple occurrences in the code where it was supposed to be a book or person. Can you make the final choice (book or person) :D

Reply:
I wanted it to be a book initially, but found out that faker doesn't have fake book titles yet so I made it a person 😞